src/isar/config/default.ini

```diff
@@ -46,8 +46,9 @@ api_port = 3000
 # Selecting a different mission planner module than local may require certain access rights
 mission_planner = local
 
-# Determines which storage module is used by ISAR
-# Options [local blob]
+# Determines which storage modules are used by ISAR
+# Comma separated list of modules to use. Each module will be called when storing results from inspections
+# Options [local blob slimm]
 # Selecting a different storage module than local may require certain access rights
 storage = local
```

Comment:
If one is to add more is it [local, some_other, another]? If so, should the default be [ local ] to make it clearer that one can add more in this manner? It is explained in the text above, but who reads that before trying?

Reply:
It is not a list as the config parser doesn't support that. It has to be a comma separated string as `local, blob, slimm`.

Can rename it to be a comma separated string

Reply:
Aha, then changing it to string would make sense

Reply:
Done

Reply:
I still see comma seperated list